.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+We are always happy to receive pull requests.
```

Comment:
this thread is not abput this topic. and yes the core team decideds what gets merged. but again: 
thisis about bug reports.